monkey/monkey_island/cc/environment/utils.py

```diff
@@ -8,25 +8,27 @@ def is_windows_os() -> bool:
 
 
 if is_windows_os():
+    import win32file
+    import win32security
```

Comment:
Do we need a `# noqa` since these imports aren't at the top? Are your pre-commit hooks running?

Reply:
We don't, isort doesn't touch them most likely because they're inside an `if` block.